Lib/fontTools/feaLib/ast.py

```diff
@@ -43,34 +46,23 @@
     'LigatureCaretByIndexStatement',
     'LigatureCaretByPosStatement',
     'LigatureSubstStatement',
-    'LookupBlock',
     'LookupFlagStatement',
     'LookupReferenceStatement',
     'MarkBasePosStatement',
-    'MarkClass',
-    'MarkClassDefinition',
-    'MarkClassName',
     'MarkLigPosStatement',
     'MarkMarkPosStatement',
     'MultipleSubstStatement',
     'NameRecord',
-    'NestedBlock',
     'OS2Field',
-    'OrderedDict',
     'PairPosStatement',
-    'Py23Error',
     'ReverseChainSingleSubstStatement',
     'ScriptStatement',
-    'SimpleNamespace',
     'SinglePosStatement',
     'SingleSubstStatement',
     'SizeParameters',
     'Statement',
```

Comment:
If `Statement` and `Expression` are in `__all__` they are presumably part of the API and should be documented.